akka-stream-tests/src/test/scala/akka/stream/DslConsistencySpec.scala

```diff
@@ -47,6 +47,7 @@ class DslConsistencySpec extends AnyWordSpec with Matchers {
     Set("equals", "hashCode", "notify", "notifyAll", "wait", "toString", "getClass") ++
     Set("productArity", "canEqual", "productPrefix", "copy", "productIterator", "productElement") ++
     Set("productElementName", "productElementNames") ++
+    Set("_1") ++
```

Comment:
Not sure where this `_1` comes from but I have an itch that tells me that if there's `_1`, then there'll be a `_2`...

Reply:
This test checks that methods that are available on javadsl are also available on the scaladsl and/or vice-versa. This collection lists methods that are allowed to exist on one but not the other. I haven't looked at the details, but apparently one of the classes now has a `_1` method on one but not the other. We haven't seen a test failure for a `_2`, so apparently not in this case ;).

Reply:
We're a test away to find out. :D

Not a show-stopper. ;-)